app/src/main/java/com/imdb_compose/ui/PersonDetailsPage.kt

```diff
@@ -0,0 +1,176 @@
+                                offsetX = 0.dp,
+                                offsetY = 0.dp,
+                                blurRadiusFilter = "SOLID"
+                            ),
+                    ) {
+                        personDetails.value?.let {
+                            Box(modifier = Modifier.padding(start = 8.dp, top = 16.dp, end = 8.dp)) {
+                                Column {
+                                    // Name & Known For
+                                    Column {
+                                        Text(
+                                            text = it.name,
+                                            fontSize = MaterialTheme.typography.headlineLarge.fontSize,
+                                            fontWeight = MaterialTheme.typography.headlineLarge.fontWeight,
+                                            fontStyle = MaterialTheme.typography.headlineLarge.fontStyle,
+                                            fontFamily = MaterialTheme.typography.headlineLarge.fontFamily
+                                        )
+                                        Spacer(modifier = Modifier.height(8.dp))
+                                        Text(
+                                            text = it.known_for_department,
+                                            fontSize = MaterialTheme.typography.titleMedium.fontSize,
+                                            fontWeight = MaterialTheme.typography.titleMedium.fontWeight,
+                                            fontStyle = MaterialTheme.typography.titleMedium.fontStyle,
+                                            fontFamily = MaterialTheme.typography.titleMedium.fontFamily
+                                        )
+                                        Spacer(modifier = Modifier.height(8.dp))
+                                    }
+                                    // Profile Pic & Bio
+                                    Row (
+                                        modifier = Modifier
+                                            .fillMaxWidth()
+                                            .height(240.dp)
+                                    ) {
+                                        Box(modifier = Modifier.width(125.dp)) {
+                                            AsyncImage(
+                                                model = "${Retrofit.BASE_IMAGE_URL}${Retrofit.IMAGE_PATH}${it.profile_path}",
+                                                contentDescription = null,
+                                                contentScale = ContentScale.FillWidth
+                                            )
+                                        }
+
+                                        Box(modifier = Modifier.padding(start = 8.dp)) {
+                                            Column {
+                                                if (it.biography != null) {
```

Comment:
Is the `biography` property not guaranteed to exist? If so, you should use the `.let` pattern you used above. You should probably not use the default `it` variable name on large `let` blocks because it be confusing what `it` refers to. Plus, then you can use it on little `let` blocks like these